src/webrtc/call.ts

```diff
@@ -480,7 +484,9 @@ export class MatrixCall extends TypedEventEmitter<CallEvent, CallEventHandlerMap
     }
 
     public get type(): CallType {
-        return this.hasLocalUserMediaVideoTrack || this.hasRemoteUserMediaVideoTrack ? CallType.Video : CallType.Voice;
+        // we may want to look for a video receiver here rather than a track to match the
+        // sender behaviour, although in practice they should be the same thing
+        return this.hasUserMediaVideoSender || this.hasRemoteUserMediaVideoTrack ? CallType.Video : CallType.Voice;
```

Comment:
The sender will be there even if the track isn't, so this could return `CallType.Video` even if we have no video in the call, afaict

Reply:
I'd argue this is correct - a video call is still a video call if everyone has their video muted, although really the lines of what is a voice and video call are very blurry.